papaya-web/src/main/java/pl/edu/agh/papaya/notification/message/Message.java

```diff
@@ -30,6 +42,17 @@ default String getMessage() {
         }
         notificationBase.append(getEnrollmentEnd().format(DATE_TIME_FORMATTER));
         notificationBase.append(".");
+
+        String sprintNotificationMessage = getSprintNotificationMessage();
+        if (sprintNotificationMessage != null) {
+            notificationBase.append("\n\n");
+            notificationBase.append(
+                    String.format(
+                            "Informacja od administratora:%n%s",
```

Comment:
Tutaj jeszcze chyba była dyskusja na temat czy na pewno taki message @jplewa ?

Reply:
`Dodatkowe informacje`/`Uwagi`/`Informacje od team leadera`? Nie wieem, ale admin brzmi jak inne stanowisko (chociaż u nas pod spodem to się tak nazywa)

Reply:
`Informacje od team leadera` 👍